recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```
@@ -24,3 +26,24 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod-tools \
     devmem2 \
 "
+RDEPENDS:${PN}-alsa = " \
+    alsa-utils \
```

Might be useful?

```suggestion
RDEPENDS:${PN}-alsa = " \
    packagegroup-base-alsa \
    alsa-utils \
```

Might need `alsa` in MACHINE_FEATURES if it isn't there. Or just copy https://git.yoctoproject.org/poky/plain/meta/recipes-core/packagegroups/packagegroup-base.bb and add

```
${@bb.utils.contains("MACHINE_FEATURES", "alsa", "packagegroup-zarhus-alsa", "", d)} \
${@bb.utils.contains("MACHINE_FEATURES", "alsa", "packagegroup-zarhus-alsa-tools", "", d)} \ 
```

to PACKAGES